### PR TITLE
update website

### PR DESCRIPTION
**Pull request by yuhanwu0302**
Hi, Steven I just updated my personal page picture.

### CODE REVIEW

**Review by stevenhwu — CHANGES_REQUESTED** [review]
can you please rename the image to `yu_han_wu`?
I try to keep the naming convention, so it will be easier fro me to manage in the long run.

File: people.md [hunk 1]
```diff
@@ -69,9 +69,9 @@ hr.rounded {
 
 
 #### [Yu-Han Wu](/personal_page/yu_han_wu)
-<img src="{{base.url}}/img/yu_han_wu.jpg" alt="Yu-Han Wu" align="left" hspace="50" width="25%">
+<img src="{{base.url}}/img/Yu-Han.jpg" alt="Yu-Han Wu" align="left" hspace="50" width="25%">
```
Comment (stevenhwu) [inline people.md]:
```suggestion
<img src="{{base.url}}/img/yu_han_wu.jpg" alt="Yu-Han Wu" align="left" hspace="50" width="25%">
```

File: personal_page/yu_han_wu.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ show_sidebar: false
 
 ## Yu-Han Wu
 
-<img src="{{site.url}}/img/yu_han_wu.jpg" align="left" hspace="10" width="25%">
+<img src="{{site.url}}/img/Yu-Han.jpg" align="left" hspace="10" width="25%">
```
Comment (stevenhwu) [inline personal_page/yu_han_wu.md]:
```suggestion
<img src="{{base.url}}/img/yu_han_wu.jpg" alt="Yu-Han Wu" align="left" hspace="50" width="25%">
```

**Review by stevenhwu — CHANGES_REQUESTED** [review]

File: people.md (hunk 1 above)
Comment (yuhanwu0302) [reply people.md]: oops, I forgot to revise this! sorry!

**Review by stevenhwu — APPROVED** [review]
Merge it please.

**Comment by yuhanwu0302** [comment]
> Merge it please.

Steven, I see this message "Only those with [write access](https://docs.github.com/articles/what-are-the-different-access-permissions) to this repository can merge pull requests."

**Pull request merged**